src/github/fetchBranches.js

```diff
@@ -35,9 +38,12 @@ module.exports = async ({ owner = github.owner, repository }) => {
 
   const {
     repository: {
+      defaultBranchRef: {
+        name
```

Comment:
Nit: If it was renamed a little bit earlier, it would be slightly easier on the eyes but it's definitely a nitpick.

```suggestion
        defaultBranch: name
```